libraries/WebServer/examples/AdvancedWebServer/AdvancedWebServer.ino

```diff
@@ -44,8 +44,10 @@ void handleRoot() {
   digitalWrite(led, 1);
   char temp[400];
   int sec = millis() / 1000;
-  int min = sec / 60;
+
   int hr = min / 60;
```

Comment:
here `hr` is set to be `min` divided by 60, but `min` is defined on the next line. This should do the trick:
```cpp
int sec = millis() / 1000;
int hr = sec / 3600;
int min = (sec / 60) % 60;
sec = sec % 60;
```